### DISCUSSION TITLE
截圖速度是否有調整上的原則

### DISCUSSION THREAD

**Discussion by bookbug666** [LmeSzinc/AzurLaneAutoScript#2123] [Q&A] [answered]
例如性能測試多少秒適合稍微接近1/遠離0.1秒之類的
想說有特別限制範圍應該是有意義的![image](https://user-images.githubusercontent.com/29662367/212596455-231b4059-3d00-4cab-88ad-386543510601.jpeg)

**Comment by LmeSzinc** [comment, marked as answer]
截图速度肯定是越快越好的，Alas 内部是禁止写 sleep 的，所以越快的截图速度意味着越快的 Alas 运行速度。但是现在电脑性能越来越高了，很多人截图耗时只有 50ms~60ms，如果无限制截图的话会带来很高的 CPU 占用，所以会限制在每 0.3s 发起一次截图操作。
